src/components/EmojiResultRow.js

```diff
@@ -0,0 +1,25 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import "./EmojiResultRow.css";
+
+export default class EmojiResultsRow extends PureComponent {
```

Comment:
Why are you using `PureComponent` instead of just `Component`

Reply:
More details: https://codeburst.io/when-to-use-component-or-purecomponent-a60cfad01a81